src/dropdown/dropdown.ts

```diff
@@ -153,13 +161,18 @@ export class NgbDropdown implements OnInit, OnDestroy {
   @Input() placement: PlacementArray;
 
   /**
-   * A selector specifying the element the dropdown should be appended to.
-   * Currently only supports "body".
-   *
-   * @since 4.1.0
-   */
+  * A selector specifying the element the dropdown should be appended to.
+  * Currently only supports "body".
+  *
+  * @since 4.1.0
+  */
   @Input() container: null | 'body';
 
+  /**
+    * Enable or disable the dynamic positioning
+    */
+  @Input() display: 'dynamic' | 'static';
```

Comment:
I agree with @benouat, the naming is confusing (even if comes straight from Bootstrap documentation)

Maybe similar to `autoClose` ?

```ts
/**
 * If `true`, positions the dropdown using `position: absolute; transform: translate()` (deafault)
 * 
 * If the dropdown is inside the `<navbar class="navbar"></navbar>`, dropdown is not positioned by default.
 * You can override positioning behaviour using this flag.
 */
@Input() autoPosition: boolean;
```

Reply:
Otherwise leave `display` as is Bootstrap documentation, but please write a proper comment :)